internal/connectors/plugins/public/bankingcircle/client/accounts.go

```diff
@@ -69,14 +67,10 @@ func (c *Client) GetAccounts(ctx context.Context, page int, pageSize int, fromOp
 
 	res := response{Result: make([]Account, 0)}
 	statusCode, err := c.httpClient.Do(req, &res, nil)
-	switch err {
-	case nil:
-		return res.Result, nil
-	case httpwrapper.ErrStatusCodeUnexpected:
-		// TODO(polo): retryable errors
-		return nil, fmt.Errorf("received status code %d for get accounts", statusCode)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get accounts, status code %d: %w", statusCode, err)
```

Comment:
I don't think you want to loose the httpwrapper error, an errors.Wrap is needed here to keep the custom error from httpwrapper

Reply:
Actually no, it's not needed since Go 1.13. As I discussed with @gfyrag yesterday, the github.com/pkg/errors is mostly obsolete (note that it is archived) as the only functionality it provides that `fmt.Errorf()` doesn't is the stack trace. Since we aren't actually making use of a stack trace downstream, we can start using `fmt.Errorf()` everywhere and try to remove the dependency of github.com/pkg/errors from this repo going forward.

Here's a good overview if you need some history of the changes since Go 1.13:
https://stackoverflow.com/questions/61933650/whats-the-difference-between-errors-wrapf-errors-errorf-and-fmt-errorf